### ISSUE TITLE
[Decision log] Don't use type-ahead search component from design system 

### ISSUE THREAD

**Issue by davidmpickett** [department-of-veterans-affairs/va.gov-cms#15545]
**EDIT - never mind - Jordan is intentionally follow a different pattern for AutoSuggest based on 526 EZ findings**

~~As we look towards converting more of the service type fields to typeahead search, would it make sense to update the community provider instance to use the Design system component?~~

## [Current implementation](https://www.va.gov/find-locations/?facilityType=urgent_care)
![Screenshot 2023-10-04 103723](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/51967950/05764eb6-acde-4f50-a4cb-0dbce7306203)

## [**Design system component**](https://design.va.gov/components/search-input#with-type-ahead)
![Screenshot 2023-10-04 103740](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/51967950/c76a56ed-5be6-4607-a27e-25c0e779bf33)

**Comment by thejordanwood** [comment]
![Screenshot 2023-10-04 122112](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/104526584/7b1defd5-adba-4b18-a3fa-e628a0cd9844)

Longer explanation for why we're aligning with the 526EZ plain language search design:

1. The type ahead and plain language searches actually function 2 different ways. The type ahead is showing the exact words/letters that are typed into the search and the search results show that at the beginning of each word. The plain language search is matching what is typed in to any part of the descriptions that we set. So in the above example, if they type in vision, that could appear at the beginning, middle, or end of the search term result. Regular/bold type probably isn't enough of a differential to show what's going on.
2. The orange highlights are used on the 526EZ form ([Sketch link](https://www.sketch.com/s/6eb1ff1a-38a5-4e15-9ff8-5ba91c800839)) and are there because people were getting confused when the search didn't have a highlight ([Relevant Github ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/30054)).
4. Laura has confirmed that the highlight passes accessibility contrast standards because it's a block shape even with grayscale, and the block stands out from everything else and indicates the item as a differentiator.
5. I kept the orange highlight because I wanted to remain consistent with the 526EZ form design, but am open to changing it to a different color.

![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/104526584/4bf9edec-e470-4a4d-bc40-6a3ebc957c40)
